blocks/alternative_mutators/plus_minus/src/field_plus.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview A field for a plus button used for mutation.
+ */
+'use strict';
+
+import * as Blockly from 'blockly/core';
+
+
+// TODO: Refactor into class?
+// TODO: Re-add plusMinus namespace prefix?
+/**
+ * Class for a plus button used for mutation.
+ * @param opt_args Arguements to pass to the 'plus' function when the button
+ *    is clicked.
+ * @constructor
+ */
+export default function FieldPlus(opt_args) {
+  this.args_ = opt_args;
+  return FieldPlus.superClass_.constructor.call(
+      this, 'media/plus.svg', 15, 15, '');
+};
+Blockly.utils.object.inherits(FieldPlus, Blockly.FieldImage);
+
+FieldPlus.fromJson = function(options) {
+  return new FieldPlus(options['args']);
+};
+
+FieldPlus.prototype.showEditor_ = function() {
+  // TODO: This is a dupe of the mutator code, anyway to unify?
```

Comment:
Can you make a function that's `doMutation(add)` that takes in a boolean to know whether to add or remove an input?

Reply:
I could, but I'm not sure that this TODO is important to fix right now.

I think this is something that should probably be fixed in core, rather than in this package, because anything that does mutation is going to look like:
```
  Blockly.Events.setGroup(true);
  var oldMutationDom = block.mutationToDom();
  var oldMutation = oldMutationDom && Blockly.Xml.domToText(oldMutationDom);

  // Do the mutating.

  var newMutationDom = block.mutationToDom();
  var newMutation = newMutationDom && Blockly.Xml.domToText(newMutationDom);
  if (oldMutation != newMutation) {
    Blockly.Events.fire(new Blockly.Events.BlockChange(
        block, 'mutation', null, oldMutation, newMutation));
  }
  Blockly.Events.setGroup(false);
```
And that doesn't just go for icons or +/- buttons, but also whatever other crazy UI people come up with.

I think a template or wrapper method that existed in core would make everything easier to maintain. But I also think that for now, leaving it like this is fine.

Idk though, whatever you think is good I'm cool with (┏－‿ ◕)┏